shared/src/sector/registered_proof.rs

```diff
@@ -130,19 +145,19 @@ impl RegisteredPoStProof {
     pub fn registered_seal_proof(self) -> Result<RegisteredSealProof, String> {
         use RegisteredPoStProof::*;
         match self {
-            StackedDRGWindow64GiBV1 | StackedDRGWinning64GiBV1 => {
+            StackedDRGWindow64GiBV1P1 | StackedDRGWindow64GiBV1 | StackedDRGWinning64GiBV1 => {
```

Comment:
Not sure what's going on here but it seems off that we're not returning V1P1 seal proofs

Reply:
Can't find any callers in builtin actors, is it possible we can just kill this function?

Reply:
Opened https://github.com/filecoin-project/ref-fvm/pull/1728 for this